.github/workflows/ci.yml

```diff
@@ -8,3 +8,161 @@ on:
+      - name: Wait for MySQL
+        if: ${{ matrix.database == 'mysql' }}
+        run: |
+          set +e
+          count=0
+          until mysqladmin ping -h 127.0.0.1 -u root --password=root --silent; do
+            if [[ "$count" == "25" ]]; then
+              exit 1
+            fi
+            (( count++ ))
+            printf '.'
+            sleep 5
+          done
+          set -e
+      - name: Wait for PostgreSQL
+        if: ${{ matrix.database == 'postgresql' }}
+        run: |
+          set +e
+          count=0
+          until $(psql -h 127.0.0.1 -U postgres -p 5432 -l > /dev/null); do
+            if [[ "$count" == "25" ]]; then
+              exit 1
+            fi
+            (( count++ ))
+            printf '.'
+            sleep 5
+          done
+          set -e
+      - name: Setup latest DDL
+        if: ${{ matrix.database == 'mysql' }}
+        run: |
+          mysql -h 127.0.0.1 -u root --password=root -e 'create database killbill'
+          ./bin/db-helper -a create --driver mysql -u root -p root -t yes -h 127.0.0.1
+      - name: Setup latest DDL
+        if: ${{ matrix.database == 'postgresql' }}
+        run: |
+          psql -h 127.0.0.1 -U postgres -p 5432 -c 'create database killbill'
+          ./bin/db-helper -a create --driver postgres -u postgres -t yes -h 127.0.0.1 --port 5432
+      - name: Install plugin specific MySQL DDL
+        if: ${{ matrix.database == 'mysql' }}
+        run: |
+          curl https://raw.githubusercontent.com/killbill/killbill-avatax-plugin/master/src/main/resources/ddl.sql | mysql -h 127.0.0.1 -u root --password=root killbill
+          curl https://raw.githubusercontent.com/killbill/killbill-payment-test-plugin/master/src/main/resources/ddl.sql | mysql -h 127.0.0.1 -u root --password=root killbill
+          curl https://raw.githubusercontent.com/killbill/killbill-email-notifications-plugin/master/src/main/resources/ddl.sql | mysql -h 127.0.0.1 -u root --password=root killbill
+      - name: Install plugin specific PostgreSQL DDL
+        if: ${{ matrix.database == 'postgresql' }}
+        run: |
+          curl https://raw.githubusercontent.com/killbill/killbill-avatax-plugin/master/src/main/resources/ddl.sql | psql -h 127.0.0.1 -U postgres -p 5432 -d killbill
+          curl https://raw.githubusercontent.com/killbill/killbill-payment-test-plugin/master/src/main/resources/ddl.sql | psql -h 127.0.0.1 -U postgres -p 5432 -d killbill
+          curl https://raw.githubusercontent.com/killbill/killbill-email-notifications-plugin/master/src/main/resources/ddl.sql | psql -h 127.0.0.1 -U postgres -p 5432 -d killbill
+      - name: Start Kill Bill
+        timeout-minutes: 4
+        run: |
+          mkdir -p /tmp/test-results
+          nohup ./bin/start-server -s > /tmp/test-results/killbill.log 2>&1 &
```

Comment:
Should we either modify our default `profiles/killbill/src/main/resources/killbill-server.properties` to have the `org.killbill.security.shiroNbHashIterations=1` or create a special one for the test?

Reply:
Also, I am confused how it works for the `postgresql' `matrix.database` - wouldn't we need to modify the `org.killbill.dao.url` property?

Reply:
> Should we either modify our default `profiles/killbill/src/main/resources/killbill-server.properties` to have the `org.killbill.security.shiroNbHashIterations=1` or create a special one for the test?

👍 for changing the default. Maybe in `work-for-release-0.23.x`?

> Also, I am confused how it works for the `postgresql` `matrix.database` - wouldn't we need to modify the `org.killbill.dao.url` property?

Yup, the PostgreSQL setup is only 80% done. I wanted to see how stable the MySQL one is for a few weeks first, before also enabling PostgreSQL.